### ISSUE TITLE
An error occurred while loading the web debug toolbar (404: Not Found).

### ISSUE THREAD

**Issue by diggersworld** [bolt/bolt#1151]
Seem to be seeing this error popup a lot recently (in a JS alert window).

```
An error occurred while loading the web debug toolbar (404: Not Found).Do you want to open the profiler?
```

Not sure what the problem is though. Having Googled it looks like a Symfony 2 thing and the general recommendation is to check the `.htaccess` file.

Mine looks the same as what's in the repo with the exception that I have a load of redirects in it:

``` htaccess
# Set the default handler.
DirectoryIndex index.php index.html index.htm

# Prevent directory listing
Options -Indexes

<FilesMatch "\.(yml|db|twig)$">
  <IfModule mod_authz_core.c>
    Require all denied
  </IfModule>
  <IfModule !mod_authz_core.c>
    Order deny,allow
    Deny from all
  </IfModule>
</FilesMatch>

<IfModule mod_rewrite.c>
  RewriteEngine on

  # START: Legacy Redirects ------------------------------------------------ #

  Redirect 301 /downloads/    /

  # END: Legacy Redirects -------------------------------------------------- #

  RewriteRule cache/ - [F]

  # Some servers require the RewriteBase to be set. If so, set to the correct folder.
  # RewriteBase /

  RewriteCond %{REQUEST_FILENAME} !-f
  RewriteCond %{REQUEST_FILENAME} !-d
  RewriteCond %{REQUEST_URI} !=/favicon.ico
  RewriteRule ^ ./index.php [L]

</IfModule>
```

**Comment by errskipower** [comment]
I am getting this error when viewing the sample site after installing Bolt on Ubuntu 15.02 running Apache2 with mod_rewrite enabled.  

Bolt is saved in a subdirectory in /var/www/ (/var/www/mysubdirectory/bolt) and I was only able to add my first user after going to http://localhost/mysubdirectory/bolt/index.php/bold/users/edit and can only view the dashboard at http://localhost/mysubdirectory/bolt/index.php/bolt.  If I remove index.php from the urls, it comes back with a 404 not found.

I am getting this popup when going to the basic sample site at http://localhost/mysubdirectory/bolt/

**Comment by xammeil** [comment]
I am also getting the same error like this , that there is an error on loading the web debug toolbar 404 page not found. How can I solve it? Can you help me?

**Comment by errskipower** [comment]
To fix my issue, I realized that I was missing the subdirectory in the rewrite.  I also ended up removing the /bolt directory so everything is in /var/www/mysubdirectory/.  In ```.htaccess```, uncomment the RewriteBase line and add your subdirectory like:

    RewriteBase /mysubdirectory/

**Comment by xammeil** [comment]
is this htaccess found in web?

**Comment by errskipower** [comment]
It's the default one that comes with installing Bolt.

**Comment by xammeil** [comment]
ahm,,, I'm  not using bolt,I'm using symfony


On 5/19/15, Emily Roth <notifications@github.com> wrote:
> It's the default one that comes with installing Bolt.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/bolt/bolt/issues/1151#issuecomment-103306363


-- 


[image: cid:image001.gif@01CF13AD.1A384900]

*Ailyn Tiongzon*

IT Cadet Technician

*TMX Philippines, Inc.*

office 63 32-230-1111 local 5555

Address

PEZA Mactan - Lapulapu City -  Cebu -  6015 - Philippines

[image: cid:image002.jpg@01CF12DD.608BF0D0]
<http://assets.timex.com/rd/fb.php>[image:
cid:image003.jpg@01CF12DD.608BF0D0] <http://assets.timex.com/rd/tw.php>

timex.com <http://assets.timex.com/rd/tx.php>

**Comment by Bill-C** [comment]
@errskipower  Thank you very much for your input on this problem. It helped me fix it. I was installing the Bolt CMS on my local and ran into the 404 error. I did not need to remove the Bolt subdirectory though. 


Seems fixed now. Thanks again.

**Comment by bigoper** [comment]
Hey All,

I'm having that error.
Only after submitting a  form (bolt-boltforms) .

Any ideas?

Thanks

**Comment by SahAssar** [comment]
@bigoper This is a 2.5 year old issue, if you are having a problem with bolt open a new issue or ask on slack with all the details, if it is with an extension open an issue with that extension, thanks :)

**Comment by bigoper** [comment]
Eventually I solved it.
But I must say that I could not get any respond on Slack.
Thanks :)

**Comment by emimarz** [comment]
I have the same problem here with silex and the profiler

**Comment by SahAssar** [comment]
@emimarz Please see my comment here, just a comment up from yours: https://github.com/bolt/bolt/issues/1151#issuecomment-293070387
